code/script.js

```diff
@@ -29,19 +42,152 @@ const showMessage = (message, sender) => {
   // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight
 }
+// Function to push to next question
+const nextQuestion = (message) => {
+  if (questionNumber === 1) {
+    userAnswer(message)
+        nameInput.value = ""
+        setTimeout(() => reply1(message), 1000)
+  }
+
+  else if (questionNumber === 2) {
+  userAnswer(message)
+  nameInput.value = ""
+        setTimeout(() => reply2(message), 1000)
+  }
+
+  else if (questionNumber === 3) {
+    userAnswer(message)
+    nameInput.value = ""
+          setTimeout(() => reply3(message), 1000)
+  }
+
+  else if (questionNumber === 4) {
+      userAnswer(message)
+      nameInput.value = ""
+            setTimeout(() => reply4(message), 1000)
+  }
+  
+  else if (questionNumber === 5) {
+        userAnswer(message)
+        nameInput.value = ""
+              setTimeout(() => reply5(message), 1000)
+  }
+
+  else if (questionNumber === 6) {
+    userAnswer(message)
+    nameInput.value = ""
+          setTimeout(() => reply6(message), 1000)
+  }
+}
 
 // Starts here
+// When website loaded, chatbot asks first question.
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage(`Hello gorgeous! What is your name?`, 'bot')
+}
+const reply1 = (message) => {
```

Comment:
Good to know this is the first reply, but too make it even easier maybe be more specific with naming this variable, such as "bookingChoice" or something similar?